core/src/main/java/org/polypheny/db/util/background/BackgroundTaskHandle.java

```diff
@@ -44,11 +44,8 @@ public BackgroundTaskHandle( String id, BackgroundTask task, String description,
 
         // Schedule
         ScheduledExecutorService exec = Executors.newSingleThreadScheduledExecutor();
-        if ( schedulingType == TaskSchedulingType.WORKLOAD ) {
-            this.runner = exec.scheduleWithFixedDelay( this, 0, 100, TimeUnit.MILLISECONDS ); // TODO MV: implement workload based scheduling
-        } else {
-            this.runner = exec.scheduleAtFixedRate( this, 0, schedulingType.getMillis(), TimeUnit.MILLISECONDS );
-        }
```

Comment:
Why have you changed this?

Reply:
You asked me to do it a while ago, if `TaskSchedulingType.WORKLOAD` was not used